Marlin/example_configurations/Felix/Configuration.h

```diff
@@ -691,7 +691,7 @@
                                  // It is assumed that when logic high = filament available
                                  //                    when logic  low = filament ran out
 #if ENABLED(FILAMENT_RUNOUT_SENSOR)
-  const bool FIL_RUNOUT_INVERTING = false; // set to true to invert the logic of the sensor.
+  #define FIL_RUNOUT_INVERTING = false; // set to true to invert the logic of the sensor.
```

Comment:
```cpp
#define BLAH_DE_BLAH = false;
```
That would result in substitution of `BLAH_DE_BLAH` with `= false;`. Probably not what we want.